recipe/build.sh

```diff
@@ -1,34 +1,36 @@
-cd src/cxx
-
-autoconf
-
-export CPATH="${PREFIX}/include"
 
-./configure --prefix=$PREFIX --enable-noisy-make
+cd src/common_libraries/libsharp
 
-# We need to manually add -fPIC because there is no way
-# to force healpix to use it otherwise. We accomplish this
-# by changing the status and then instantiating it
+./configure --prefix=$PREFIX --disable-silent-rules --disable-dependency-tracking
 
 if [ "$(uname)" == "Darwin" ]; then
-    
+
```

Comment:
I think this block needs to cover the main healpix_cxx build below it too?

Reply:
Is it needed since I have added `--enable-shared`, to the configuring of the healpix_cxx, which adds the `-fPIC` flag.

Reply:
IDK. If -fPIC is in the flags in the logs then it is fine.

Reply:
I just checked and I confirm I see `-fPIC` in the logs. I also found https://stackoverflow.com/questions/629961/how-can-i-set-ccshared-fpic-while-executing-configure, confirming this.